roborazzi-junit-rule/src/main/java/com/github/takahirom/roborazzi/RoborazziRule.kt

```diff
@@ -157,9 +177,18 @@ class RoborazziRule private constructor(
     description: Description,
     captureRoot: CaptureRoot
   ) {
-    val evaluate = {
+    val evaluate: () -> Unit = {
       try {
+        val accessibilityChecks = options.accessibilityChecks
+        // TODO enable a11y before showing content
```

Comment:
May I ask the reason why we enable this before showing content and why we cannot enable it?

Reply:
I think there is a mode where it would run throughout the test with multiple screenshots.

We would need to hook it in to call it at those points. 

But doing it once at the end seemed safer to start with.

Suggestions?

Reply:
One example I've seen is screenshotting on each click for interaction tests.

Reply:
Should this and the later problem be solved by running during each screenshotting?

Reply:
Yes, we have several options to implement, such as checking after the test, verifying each `captureRoboImage()` call, checking every layout, or creating `checkRoboAccessibility()`, among others. 
I don't think using every layout is cost-effective. 
Sorry, I'm just considering which option is best.

Reply:
Is there anything you need from me, or should I leave these further changes to you?

Reply:
Thanks, and sorry for being late. I have a presentation at the end of this month. I checked the code, and `processOutputImageAndReport` doesn't have a view or anything. It's a bit difficult to integrate accessibility checks into `captureRoboImage`. So, I added `checkRoboAccessibility` and renamed `AccessibilityChecksValidate` to `ValidateAfterTest`. Could you check it out? 
https://github.com/takahirom/roborazzi/commit/ef6a2657790ad48e0c517758216e99a54bef6afc

Reply:
no need to apologise, thanks for spending the time to review this. Take your time.

Reply:
ValidateAfterTest makes sense.

Reply:
I wanted to provide accessibility options, such as RoborazziOptions, for use in checkRoboAccessibility. Therefore, I changed the structure of the code and refactored the naming conventions.

https://github.com/takahirom/roborazzi/pull/557/files/ef6a2657790ad48e0c517758216e99a54bef6afc..2032a4f820d35976288ea0f2f2791b5f1d82a693

```kotlin
fun SemanticsNodeInteraction.checkRoboAccessibility(
  roborazziATFAccessibilityCheckOptions: RoborazziATFAccessibilityCheckOptions = provideATFAccessibilityOptionsOrCreateDefault(),
  roborazziOptions: RoborazziOptions = provideRoborazziContext().options,
) {
  roborazziATFAccessibilityCheckOptions.checker.runAccessibilityChecks(
    checkNode = RoborazziATFAccessibilityChecker.CheckNode.Compose(this),
    roborazziOptions = roborazziOptions,
    failureLevel = roborazziATFAccessibilityCheckOptions.failureLevel,
  )
}
```